accounting/accounting_cli_functions.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python3
+
```

Comment:
The executable bit for this python file was off when I try this under Docker. 

`chmod u+x accounting_cli_functions.py`